atd-vzv/src/constants/nav.js

```diff
@@ -5,12 +5,6 @@ import React from "react";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faChartBar, faMap } from "@fortawesome/free-solid-svg-icons";
 
-// Detect the environment
-export const isDevelopment =
-  !process.env.NODE_ENV || process.env.NODE_ENV === "development";
-
-export const isPreview = process.env.REACT_APP_VZV_ENVIRONMENT === "PREVIEW";
-
```

Comment:
If I understood correctly in dev sync, I dont think we need this distinction anymore and we are just gonna pull production socrata data going forward whether or not we are in development?

Reply:
This makes sense to me so we don't trip over this when running locally going forward. I think that we can decide how to point the app to staging when/if https://github.com/cityofaustin/atd-data-tech/issues/13087 is prioritized.

Reply:
agreed.